src/api/organization.go

```diff
@@ -189,6 +189,17 @@ func (a *OrganizationAPI) DeleteOrganization(c *gin.Context) {
 
 	user := auth.GetCurrentUser(c.Request)
 	organization, err := auth.GetOrganizationById(uint(id))
+	if err != nil {
+		message := "error getting organization: " + err.Error()
+		log.Info(message)
+		statusCode := auth.GormErrorToStatusCode(err)
+		c.AbortWithStatusJSON(statusCode, common.ErrorResponse{
+			Code:    statusCode,
+			Message: message,
+			Error:   message,
+		})
+	}
+
```

Comment:
**Note**: I think it is debatable whether we want to return an error on trying to delete a non-existent entity or just return a success, because the entity "is deleted" already.

Reply:
I guess the error here could be any kind of GORM error, like the connection to the database lost, which would be good to get... but if a `_` in place of the `err` would suffice here, I can change it, no problem 🙂

Reply:
We can leave it as is from me, were just curious whether we thought this through, but it is fine for me.